torch/distributed/_composable/fsdp/fully_shard.py

```diff
@@ -247,3 +274,14 @@ def _apply(self, *args: Any, **kwargs: Any) -> Any:
                     : fsdp_param.sharded_size[0]
                 ]
         return ret
+
+
+class UnshardHandle:
```

Comment:
may need to add some docs on this class since it's a public API?

Reply:
I think docs are not rendered yet since this is in a private folder, but let me add some docs.